solutions/swb-reference/README.md

```diff
@@ -6,348 +6,728 @@ The 0.1.0 beta release is available for customers to test the newly re-architect
 
-## Set Up
+Note: For Solutions Implementations deployment, the `awsRegionShortName` value will be pre-assigned to `test`.
 
-1. In root directory at `solution-spark-on-aws` run `rush install`
-2. Copy `src/config/example.yaml` and create a new file in the format `<STAGE>.yaml` in the config folder
-3. Uncomment the `stage` attribute and provide the correct `<STAGE>` value for the attribute
-4. Uncomment `awsRegion` and `awsRegionShortName`. `aws-region` value can be one of the values on this [table](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html#Concepts.RegionsAndAvailabilityZones.Regions), under the `Region` column. `awsRegionName` can be a two or three letter abbreviation for that region, of your own choosing.
-5. Uncomment `rootUserEmail` and provide the main account user's email address
-6. Run `chmod 777 <STAGE>.yaml` to allow local script to read the file
+### Setup CDK
+We'll be using AWS CDK to deploy our code to AWS. Follow the steps below to onboard CDK onto your AWS `Main Account`.
 
-Note: For Solutions Implementations deployment, the `awsRegionShortName` value will be pre-assigned to `test`.
+In `swb-reference` root directory run the follow code
+```bash
+rush install
+rush build
+rushx compile
+STAGE=<STAGE> rushx cdk bootstrap
+```
 
-## Running Code Locally
+After bootstrap is completed you'll see a message like this
+```bash
+Found configuration in /Users/<user>/workplace/solution-spark-on-aws/rush.json
 
-If you have made changes to the `environment` package or the `swb-reference` package follow these steps
+Rush Multi-Project Build Tool 5.62.1 - Node.js 14.17.0 (LTS)
+> "cdk bootstrap"
 
-1. In `solution-spark-on-aws` root directory run `rush build`
-2. In `solution-spark-on-aws/solutions/swb-reference` root directory run `STAGE=<STAGE TO RUN LOCALLY> ./scripts/runLocally.sh`. This will run a local lambda server.
+ ⏳  Bootstrapping environment aws://123456789012/ap-northeast-3...
+Trusted accounts for deployment: (none)
+Trusted accounts for lookup: (none)
+Using default execution policy of 'arn:aws:iam::aws:policy/AdministratorAccess'. Pass '--cloudformation-execution-policies' to customize.
+CDKToolkit: creating CloudFormation changeset...
+ ✅  Environment aws://123456789012/ap-northeast-3 bootstrapped.
+```
 
-## Deploying Code
+### Deploy the code
+In `swb-reference` root directory run the following code
 
-Run one time to Bootstrap the CDK
+```bash
+STAGE=<STAGE> rushx cdk-deploy              # Deploy code to `Main Account` on AWS
+```
+Take note of the following Cloudformation outputs. We will be using them in future steps.
+```
+S3BucketArtifactsArnOutput
+AccountHandlerLambdaRoleOutput
+ApiLambdaRoleOutput
+StatusHandlerLambdaRoleOutput
+APIGatewayAPIEndpoint
+```
 
-`STAGE=<STAGE> rushx cdk bootstrap`
+Run the post deployment step
+```
+STAGE=<STAGE> rushx run-postDeployment      # Setup Service Catalog portfolio and products
```

Comment:
What's the "# Setup Service Catalog portfolio and products"?

Reply:
It is just specifying that running post deployment script will setup the portfolio and products so we have the environment type templates